docs/runtime-contract.md

```diff
@@ -125,47 +112,36 @@ the OCI specification as long as:
 - Platforms MAY provide mechanisms for post-mortem viewing of filesystem
   contents from a particular execution. Because containers (particularly failing
   containers) can experience frequent starts, operators or platform providers
-  SHOULD limit the total space consumed by these failures.
-- A container should write its own termination message to `/dev/termination-log`
-  by default. If no message is written by the container, the last few lines of
-  log output should be reported as the execution error (i.e. by
-  [setting the `terminationMessagePolicy` to `FallbackToLogsOnError`](https://kubernetes.io/docs/tasks/debug-application-cluster/determine-reason-pod-failure/#customizing-the-termination-message))
-  on Kubernetes.
-
-### Warnings
-
-As specified by OCI.
+  MAY limit the total space consumed by these failures.
 
 ### Operations
 
 [The OCI interface](https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc3/runtime.md#operations)
 SHOULD NOT be exposed within the container. The operator or platform provider
-MAY have the ability to directly interact with the OCI interface, but that is
-beyond the scope of this specification.
+MAY have the ability to directly interact with the OCI interface.
```

Comment:
@evankanderson @dgerd did we want to reword the OCI interface parts?

Reply:
I think Dan was saying that we could probe at some well-known locations, but couldn't definitively guarantee that this was not exposed.

I'm fine with leaving as-is or moving to the developer guide.